server/app/controllers/meeting.js

```diff
@@ -121,7 +124,8 @@ router.post('/invite', function(req, res, next) {
         responseLink: 'http://ezpick.herokuapp.com/meeting/' + meeting.id,
       }
     );
-
+    newEmails = meeting.invited.concat(newEmails);
```

Comment:
On this line, `newEmails` stops being the new emails, which is confusing. You can simply do `meeting.invited = meeting.invited.concat(newEmails)`